backend/telemetry_core/src/find_location.rs

```diff
@@ -43,7 +43,7 @@ where
     cache.insert(
         Ipv4Addr::new(127, 0, 0, 1).into(),
         Arc::new(NodeLocation {
-            latitude: 52.516_6667,
+            latitude: 52.516_666,
```

Comment:
can you explain this change?

Reply:
It was fixed by clippy automatically, but originally it was this [lint](https://rust-lang.github.io/rust-clippy/master/index.html#inconsistent_digit_grouping)

Reply:
Tried to fix it with `52.516_666_6` and clippy warned me with [this](https://rust-lang.github.io/rust-clippy/master/index.html#excessive_precision)

Reply:
ah ok, is `latitude` a `f32` then? Otherwise it makes no sense :D

Reply:
Yeah, it is!